doku.php

```diff
@@ -11,6 +11,16 @@
 // update message version - always use a string to avoid localized floats!
 $updateVersion = "50";
 
+// let people know if they are running an unsupported version of PHP
+$required_php = "5.6";
```

Comment:
This should probably be a constant and be reused in inc/info.php where we also check for the PHP version.